test/stream.browser.spec.ts

```diff
@@ -1,74 +1,68 @@
 import * as underTest from '../src/stream'
 import { expect, assert } from 'chai'
 
+function setup (): { peerConnection: RTCPeerConnection, datachannel: RTCDataChannel, webrtcStream: underTest.WebRTCStream } {
+  const peerConnection = new RTCPeerConnection()
+  const datachannel = peerConnection.createDataChannel('whatever', { negotiated: true, id: 91 })
```

Comment:
why not make the data channel id optional and generate a random ID by default? Seems better to me than having to invent strings like this?